modules/file-input/test/spec.coffee

```diff
@@ -0,0 +1 @@
+describe "File Input", ->
```

Comment:
Yeah, I plan to add some - although unfortunately there's no way to 'set' the files to use due to security issues so it's going to be fun to test everything...